accord-core/src/main/java/accord/impl/SimpleProgressLog.java

```diff
@@ -500,6 +500,7 @@ void run(Command command)
                     Unseekables<?, ?> someKeys = unseekables(command);
 
                     BiConsumer<Known, Throwable> callback = (success, fail) -> {
+                        if (fail != null) node.agent().onUncaughtException(fail);
```

Comment:
I have a lot of mixed feelings about this... right now we change our state to `Expected` and no-op if there is a failure, but since we do not handle I add this.  We see very frequent timeouts which caused BurnTest to fail 90% of the time, so I added a filter to ignore Timeout... which feels off...

The main reason I added this is that non-timeout exceptions were happening and we were ignoring them, so we were finding issues and by not calling `onUncaughtException` we wouldn't notice them.